clients/drcachesim/tracer/syscall_pt_trace.cpp

```diff
@@ -172,9 +177,9 @@ syscall_pt_trace_t::stop_syscall_pt_trace()
     cur_recording_sysnum_ = -1;
 
     /* Reset the pttracer handle for next syscall.
-     * TODO i#5505: To reduce the overhead caused by pttracer initialization, we need to
-     * share the same pttracer handle for all syscalls per thread. And also, we need to
-     * improve the libpt2ir to support streaming decoding.
+     * TODO i#5505: To reduce the overhead caused by pttracer initialization, we
+     * need to share the same pttracer handle for all syscalls per thread. And
+     * also, we need to improve the libpt2ir to support streaming decoding.
```

Comment:
Not seeing any change here?  Accidental reformat?

Reply:
Not sure. I assumed it was clang-format's doing.

Reply:
Reverted

Reply:
That's probably what happened yeah.